controllers/cartController.js

```diff
@@ -0,0 +1,94 @@
+const cartService = require("../services/cartService");
+const etc = require("../utils/etc");
+const secretKey = process.env.SECRET_KEY;
+
+const addCartItem = async (req, res) => {
+  try {
+    const acccesToken = req.headers.authorization;
+    const decoded = etc.decoded(acccesToken, secretKey);
+    const userId = decoded.userId;
+
+    const productId = req.params.productId;
+    const quantity = parseInt(req.body.quantity);
+
+    if (!productId || !quantity) {
+      return res.status(400).json({ message: "KEY ERROR" });
+    }
```

Comment:
```suggestion
    if (!productId || !quantity) {
      const error = new Error("KEY_ERROR");
      error.statusCode = 400;
      throw error
    }
```
- 에러 곧바로 리턴 보다는 throw를 이용해서 catch 에서 에러가 return 되도록 해주세요 :)